### ISSUE TITLE
about the pretrain checkpoint filw

### ISSUE THREAD

**Issue by dingjiangang** [TropComplique/light-head-rcnn#5]
i have a question ,how can i get the pretrained/shufflenet_v2/model.ckpt-1661328' file  ,thank you

**Comment by TropComplique** [comment]
You can get it here:
https://github.com/TropComplique/shufflenet-v2-tensorflow#shufflenet-v2
(it is 1.0x version).

**Comment by dingjiangang** [comment]
thank you, any other pretrained model could be available? for example you mentioned methods,
 if params['backbone'] == 'resnet':
        feature_extractor = resnet
        checkpoint_scope = 'resnet_v1_50/'
    elif params['backbone'] == 'mobilenet':
        feature_extractor = lambda x: mobilenet(x, params['depth_multiplier'])
        checkpoint_scope = 'MobilenetV1/'
    elif params['backbone'] == 'shufflenet':
